runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java

```diff
@@ -227,28 +222,19 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 	 */
 	private ListType checkScamList(String nameToBeChecked)
 	{
-		if (wdrScamArrayList.size() > 0 && this.enableWDR)
+		if (wdrScamArrayList.size() > 0 && this.enableWDR && wdrScamArrayList.stream().anyMatch(nameToBeChecked::equalsIgnoreCase))
```

Comment:
The name collections should be HashSets, so you can just use .contains() to look them up as quick as you can